src/styles/SurveyComponents.styles.scss

```diff
@@ -0,0 +1,91 @@
+.survey-next {
+  margin: 1rem;
+}
+.survey-back {
+  margin: 1rem;
+  background: #faf9f9;
+  color: var(--accent-color);
```

Comment:
I couldn't find this css variable anywhere in the PR, also, shouldn't you use the sass variable convention instead of the css convention here?

Reply:
ok I'm honestly not sure where it's defined,  I saw it used here
https://github.com/egytech-fyi/egytech-fyi/blob/10ce8fffbb0e21ce0c3106e2459f7bdfa2c7a823/src/styles/Footer.styles.scss#L24)
and removing it does change the color 🤷 but I removed it from the PR just in case